components/command/windowsOSCommand.go

```diff
@@ -54,6 +54,10 @@ func (fs windowsOSCommand) GetTemporaryDirectory() string {
 	return "$env:TEMP"
 }
 
+func (fs windowsOSCommand) GetHomeDirectory() string {
+	return "$env:HOMEPATH\\$env:HOMEPATH"
```

Comment:
Did you mean `"$env:HOMEPATH"`?

Reply:
Oh thank you, I meant `$env: HOMEDRIVE$env:HOMEPATH`, good catch !